doc/source/user/basics.interoperability.rst

```diff
@@ -314,6 +315,23 @@ data type.
    <https://github.com/pytorch/pytorch/blob/master/torch/overrides.py>`__
    for details.
 
+Note also that we can see ``__array_wrap__`` in action here, even though
+``torch.Tensor`` is not a subclass of ndarray::
+
+   >>> import torch
+   >>> t = torch.arange(4)
+   >>> np.abs(t)
+   tensor([0, 1, 2, 3])
+   >>> wrap = lambda self, array: np.asarray(array)
```

Comment:
From this line on it's best to delete the example (only keep until `tensor([0, 1, 2, 3])`). I included it to show you that it really was `__array_wrap__` that is responsible for creating the `tensor` return type, but now the example looks like this is somehow normal usage.

Reply:
Should the docstring get a pragma to be ignored in doctests? I don't think we want to import tensorflow

Reply:
I have added this page to the refguide exclude list as none of it is very useful to run as doctests.